API/hermes_abi/README.md

```diff
@@ -0,0 +1,5 @@
+# Experimental C based stable ABI for Hermes
+
+This directory contains ongoing work to develop a stable C-based ABI for Hermes and an accompanying C++ JSI wrapper. It is a work in progress and is not supported for general use.
+
+The goal of this ABI is to allow Hermes to be updated independently of the rest of a React Native application. Note that this does not solve the general problem of ABI stability for RN extensions, since they are still written against the C++ JSI, and consume a C++ `jsi::Runtime` provided by RN.
```

Comment:
does that mean that the hermes bytecode will also be made stable at some point so that hermes can be updated independently of React Native app?

Reply:
Great question. Bytecode stability is not currently planned, but is certainly on our mind for the next major version of Hermes. Note that it is not strictly necessary for swapping Hermes, since you can also swap the compiler binary for a newer release.